_data/team.json

```diff
@@ -0,0 +1,158 @@
+            "name": "Michael Ficarra",
+            "website": "https://twitter.com/jspedant",
+            "avatar_url": "https://avatars1.githubusercontent.com/u/218840?v=4"
+        },
+        {
+            "username": "xjamundx",
+            "name": "Jamund Ferguson",
+            "website": "http://www.jamund.com",
+            "avatar_url": "https://avatars3.githubusercontent.com/u/246143?v=4"
+        },
+        {
+            "username": "mikesherov",
+            "name": "Mike Sherov",
+            "website": "",
+            "avatar_url": "https://avatars1.githubusercontent.com/u/364532?v=4"
+        },
+        {
+            "username": "hzoo",
+            "name": "Henry Zhu",
+            "website": "https://www.henryzoo.com",
+            "avatar_url": "https://avatars1.githubusercontent.com/u/588473?v=4"
+        },
+        {
+            "username": "vitorbal",
+            "name": "Vitor Balocco",
+            "website": "https://twitter.com/vitorbal",
+            "avatar_url": "https://avatars2.githubusercontent.com/u/626038?v=4"
+        },
+        {
+            "username": "zxqfox",
+            "name": "Alexej Yaroshevich",
+            "website": "http://about.me/yaroshevich",
+            "avatar_url": "https://avatars3.githubusercontent.com/u/677518?v=4"
+        },
+        {
+            "username": "JamesHenry",
+            "name": "James Henry",
+            "website": "https://jameshenry.blog",
+            "avatar_url": "https://avatars1.githubusercontent.com/u/900523?v=4"
+        },
+        {
+            "username": "markelog",
+            "name": "Oleg Gaidarenko",
+            "website": "https://twitter.com/arkel",
+            "avatar_url": "https://avatars0.githubusercontent.com/u/945528?v=4"
+        },
+        {
+            "username": "soda0289",
+            "name": "Reyad Attiyat",
+            "website": "",
+            "avatar_url": "https://avatars1.githubusercontent.com/u/2373964?v=4"
+        },
+        {
+            "username": "VictorHom",
+            "name": "Victor Hom",
+            "website": "https://twitter.com/victorhomburger",
+            "avatar_url": "https://avatars1.githubusercontent.com/u/3211873?v=4"
+        },
+        {
+            "username": "IanVS",
+            "name": "Ian VanSchooten",
+            "website": "http://devnull.guru",
+            "avatar_url": "https://avatars0.githubusercontent.com/u/4616705?v=4"
+        },
+        {
+            "username": "pedrottimark",
+            "name": "Mark Pedrotti",
+            "website": "http://twitter.com/ittordepam",
+            "avatar_url": "https://avatars1.githubusercontent.com/u/11862657?v=4"
+        }
+    ],
+    "committers": [
+        {
+            "username": "Aladdin-ADD",
+            "name": "薛定谔的猫",
+            "website": "",
+            "avatar_url": "https://avatars2.githubusercontent.com/u/13050025?v=4"
+        },
+        {
+            "username": "eslintbot",
```

Comment:
I'm making a "request changes" review as a reminder to regenerate this list, which will have the effect of removing `eslintbot`. Feel free to dismiss this review and merge after `eslintbot` has been removed.